### PR TITLE
Add biotools for iqtree

### PR DESCRIPTION
**Pull request by bebatut**
FOR CONTRIBUTOR:
* [X] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [X] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/iqtree/iqtree.xml [hunk 1]
```diff
@@ -928,9 +934,8 @@ Running example
 ---------------------
 
 From the download_ there is an example alignment called `example.phy`
-in PHYLIP format. This example contains parts of the mitochondrial DNA sequences of several animals (Source: `Phylogenetic Handbook`_)
+in PHYLIP format. This example contains parts of the mitochondrial DNA sequences of several animals (Source: The Phylogenetic Handbook)
```
Comment (bernt-matthias) [inline tools/iqtree/iqtree.xml:937]:
```suggestion 
in PHYLIP format. This example contains parts of the mitochondrial DNA sequences of several animals.
```

**Review by bernt-matthias — APPROVED** [review]
Looks good to me. Up to you if you take my suggestion.

**Review by bernt-matthias — APPROVED** [review]

**Comment by bernt-matthias** [comment]
Hrm: 

```
Job in error state.. tool_id: iqtree, exit_code: 2, stderr: ERROR: Alignment sequence LngfishAu does not appear in the tree
ERROR: Tree taxa and alignment sequence do not match (see above)
```

**Comment by bebatut** [comment]
The tests are failing now... Not sure why (not failing locally). Investigating

**Comment by bgruening** [comment]
@bebatut does this rebase makes sense to you?

**Comment by bebatut** [comment]
Yes :)

**Comment by bebatut** [comment]
Thanks @bernt-matthias and @bgruening

**Pull request merged**